sensors/video/sensorhub-driver-axis/build.gradle

```diff
@@ -6,9 +6,9 @@ dependencies {
   implementation 'org.sensorhub:sensorhub-core:' + oshCoreVersion
   implementation project(':sensorhub-driver-rtpcam')
   embeddedImpl('net.sf.jipcam:jipcam:0.9.1') {
-    exclude group: 'javax.servlet', module: 'servlet-api'
```

Comment:
I think we can keep the excludes because we don't use thess parts of the jipcam library I think

Reply:
Keeping these excludes is causing an exception:

```sh
org.osgi.framework.BundleException: Unable to resolve org.sensorhub.sensorhub-driver-axis [34](R 34.0): missing requirement [org.sensorhub.sensorhub-driver-axis [34](R 34.0)] osgi.wiring.package; (osgi.wiring.package=org.apache.commons.cli) Unresolved requirements: [[org.sensorhub.sensorhub-driver-axis [34](R 34.0)] osgi.wiring.package; (osgi.wiring.package=org.apache.commons.cli)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4398)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2308)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:1007)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:993)
	at org.sensorhub.impl.osgi.OsgiLauncher.installBundles(OsgiLauncher.java:161)
	at org.sensorhub.impl.osgi.OsgiLauncher.main(OsgiLauncher.java:267)
```

Either, we exclude these packages, and we have to add more lines into Import-Package, or we don't exclude.